rest-service/manager_rest/resource_manager.py

```diff
@@ -1173,9 +1174,14 @@ def create_deployment(self,
                           visibility,
                           inputs=None,
                           bypass_maintenance=None,
-                          skip_plugins_validation=False):
+                          skip_plugins_validation=False,
+                          site_name=None):
         blueprint = self.sm.get(models.Blueprint, blueprint_id)
         plan = blueprint.plan
+        site = None
+        if site_name:
```

Comment:
maybe change to one liner
site = self.sm.get(models.Site, site_name) if site_name else None